app/classes/Transvision/Utils.php

```diff
@@ -233,13 +233,12 @@ public static function getRepoStrings($locale, $repository)
 
         $file = TMX . "{$locale}/cache_{$locale}_{$repository}.php";
 
-        if (! $tmx = Cache::getKey($file)) {
-            if (is_file($file)) {
-                include $file;
-                Cache::setKey($file, $tmx);
-            }
+        if (! is_file($file)) {
+            return [];
         }
 
+        include $file;
+
         return $tmx !== false ? $tmx : [];
```

Comment:
If the file is broken we should probably get a warning for `$tmx` not being defined (before it would have been at least false because of Cache::getKey), so I think it makes sense to test if it's set?